rules/prefer-string-raw.js

```diff
@@ -64,6 +61,52 @@ const create = context => {
 			},
 		};
 	});
+
+	context.on('TemplateLiteral', node => {
+		if (node.parent.type === 'TaggedTemplateExpression') {
+			return;
+		}
+
+		let unescaped = '';
+		let hasBackslash = false;
+
+		for (let index = 0; index < node.quasis.length; index++) {
+			const quasi = node.quasis[index];
+			const {raw, cooked} = quasi.value;
+
+			if (cooked.at(-1) === BACKSLASH) {
+				return;
+			}
+
+			const unescapedQuasi = unescapeBackslash(raw);
+			if (unescapedQuasi !== cooked) {
+				return;
+			}
+
+			if (cooked.includes(BACKSLASH)) {
+				hasBackslash = true;
+			}
+
+			if (index > 0) {
+				unescaped += '${' + context.sourceCode.getText(node.expressions[index - 1]) + '}';
+			}
+
+			unescaped += unescapedQuasi;
```

Comment:
We don't need concat all the string, just save the replacements and replace `TemplateElement`s one by one.

For example, current fix are removing spaces and comments in `expression`s.